docs/library/espnow.rst

```diff
@@ -739,7 +753,9 @@ point. When an ESP32 or ESP8266 device connects to a Wifi Access Point (see
 `ESP32 Quickref <../esp32/quickref.html#networking>`__) the following things
 happen which affect ESPNow communications:
 
-1. Wifi Power-saving Mode is automatically activated and
+1. Wifi Power-saving Mode (
+   `sta.config(pm=sta.PM_PERFORMANCE<network.WLAN.PM_PERFORMANCE>`)
```

Comment:
I'm not sure what's going on here, but it should be either `sta.PM_PERFORMANCE` or `network.WLAN.PM_PERFORMANCE`.

Note that 53cb073571196d2ba7174706d5d784e42e5430cf was merged so the latter will work.